manifests/00-custom-resource-definition.yaml

```diff
@@ -17,3 +17,187 @@ spec:
+                    is currently supported only on the AWS platform.  * HostNetwork  Publishes
+                    the ingress controller on node ports where the ingress controller
+                    is deployed.  In this configuration, the ingress controller deployment
+                    uses host networking, bound to node ports 80 and 443. The user
+                    is responsible for configuring an external load balancer to publish
+                    the ingress controller via the node ports.  * Private  Does not
+                    publish the ingress controller.  In this configuration, the ingress
+                    controller deployment uses container networking, and is not explicitly
+                    published. The user must manually publish the ingress controller.'
+                  type: string
+              required:
+              - type
+              type: object
+            namespaceSelector:
+              description: namespaceSelector is used to filter the set of namespaces
+                serviced by the ingress controller. This is useful for implementing
+                shards.  If unset, the default is no filtering.
+              type: object
+            nodePlacement:
+              description: nodePlacement enables explicit control over the scheduling
+                of the ingress controller.  If unset, defaults are used. See NodePlacement
+                for more details.
+              properties:
+                nodeSelector:
+                  description: 'nodeSelector is the node selector applied to ingress
+                    controller deployments.  If unset, the default is:    beta.kubernetes.io/os:
+                    linux   node-role.kubernetes.io/worker: ''''  If set, the specified
+                    selector is used and replaces the default.'
+                  type: object
+              type: object
+            replicas:
+              description: replicas is the desired number of ingress controller replicas.
+                If unset, defaults to 2.
+              format: int32
+              type: integer
+            routeSelector:
+              description: routeSelector is used to filter the set of Routes serviced
+                by the ingress controller. This is useful for implementing shards.  If
+                unset, the default is no filtering.
+              type: object
+          type: object
+        status:
+          description: status is the most recently observed status of the IngressController.
+          properties:
+            availableReplicas:
+              description: availableReplicas is number of observed available replicas
+                according to the ingress controller deployment.
+              format: int32
+              type: integer
+            conditions:
+              description: 'conditions is a list of conditions and their status.  Available
```

Comment:
I didn't see `status.conditions` when running `oc get ingresscontrollers.operator.openshift.io/default -n openshift-ingress-operator -o yaml`.

Reply:
@lihongan status conditions are being implemented in PR #174, which is expected to merge very soon.

Reply:
Get it, thank you @danehans